overcloud.yml

```diff
@@ -2,38 +2,57 @@
 - hosts: undercloud
   gather_facts: yes
   tasks:
-      # RHBZ 1756492
-      - name: exclude ceph images
-        lineinfile:
-          path: /home/stack/containers-prepare-parameter.yaml
-          insertafter: "- push_destination: true"
-          line: '    excludes:'
-        when: osp_release > 13
-      - name: exclude ceph images
-        lineinfile:
-          path: /home/stack/containers-prepare-parameter.yaml
-          insertafter: "    excludes:"
-          line: '    - ceph'
-        when: osp_release > 13
-      - name: get tht version
-        shell: |
-          grep  "heat_template_version" /usr/share/openstack-tripleo-heat-templates/overcloud.j2.yaml | cut -d ':' -f2
-        register: tht_version
-      - name: set heat_template_version
-        set_fact:
-          heat_template_version: "{{ tht_version.stdout }}"
-      - name: generate firstboot-nvme.yaml
-        template:
-          src: "firstboot-nvme.yaml.j2"
-          dest: "/home/stack/firstboot-nvme.yaml"
-        when: passthrough_nvme is defined or mount_nvme is defined
+    - block:
+        # RHBZ 1756492
+        - name: exclude ceph images
+          lineinfile:
+            path: /home/stack/containers-prepare-parameter.yaml
+            insertafter: "- push_destination: true"
+            line: '    excludes:'
+          when: osp_release > 13
```

Comment:
nit: this check can be moved to block level

Reply:
Done